### PR TITLE
update yml file - try paths for triggering CI for unit test

### PR DESCRIPTION
**Pull request by Boopster**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ name: Continuous Integration
 # On every pull request on the develop branch, execute all tests [CI]
 on:
   pull_request:
-    branches:
-      - develop
```
Comment (stephenbit) [inline .github/workflows/ci.yml:7]: Should we remove `develop` now that it has been deleted and we aren't using it?

Comment (Boopster) [reply .github/workflows/ci.yml:7]: I think you have clicked on the file showing the deleted lines

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@ name: Continuous Integration
 # On every pull request on the develop branch, execute all tests [CI]
 on:
   pull_request:
```
Comment (mikemherron) [inline .github/workflows/ci.yml:5]: Think this looks good - I'm not super familiar with GitHub actions, but would it be simple to update this to also run all tests on every commit, as well as on each pull request? Most times I've worked with CI in the past it's usually been on each commit.

Comment (Boopster) [reply .github/workflows/ci.yml:5]: I will have a look.

**Pull request merged**